src/matchers/toBeAccessible/index.spec.ts

```diff
@@ -109,4 +109,21 @@ test.describe.parallel('toBeAccessible', () => {
     expect(duration).toBeGreaterThan(1000)
     expect(duration).toBeLessThan(1500)
   })
+
+  test('should allow providing custom report filename', async ({ page }) => {
+    test.fail()
+    const reportFileName = 'custom-report.html'
+    const content = await readFile('inaccessible.html')
+    await page.setContent(content)
+    await expect(page).toBeAccessible({ timeout: 2000, reportFileName })
+    await expect(reportFileExists(test.info(), reportFileName)).toBe(true)
```

Comment:
This line will never be called since the test will fail on the call to `toBeAccessible`.  What you could do is this to swallow the error.

```js
await expect(page).toBeAccessible({ timeout: 2000, reportFileName }).catch(() => {})
```